common/app/common/pages/helpers.cljc

```diff
@@ -408,12 +408,18 @@
     parent-idx))
 
 (defn split-path
-  [path]
   "Decompose a string in the form 'one / two / three' into
-  an array of strings, normalizing spaces."
+  a vector of strings, normalizing spaces."
+  [path]
   (->> (str/split path "/")
```

Comment:
This can be done using transducers for better efficiency.

Reply:
One day I'll definitively learn what transducers are, and what are they useful for :P :P :P :P